curvefs/src/client/fuse_s3_client.h

```diff
@@ -92,6 +101,18 @@ class FuseS3Client : public FuseClient {
  private:
     // s3 adaptor
     std::shared_ptr<S3ClientAdaptor> s3Adaptor_;
+
+    Thread bgFetchThread_;
+    std::atomic<bool> bgFetchStop_;
```

Comment:
Do not define private members and functions together; use a new class to manage them.

Reply:
got it